components/raftstore-v2/src/operation/command/admin/merge/prepare.rs

```diff
@@ -507,6 +534,8 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
                 if pending_peers.is_empty() && flushed {
                     Ok(true)
                 } else {
+                    // Retry send_check_trim_status in case messages are dropped.
+                    self.send_check_trim_status(store_ctx);
```

Comment:
Would it lead to sending too many useless requests when the trim is just slow but not lost?
how about adding a timeout threshold and only retry when it expires?

Reply:
Can leverage the start_time in WaitForTrimStatus { pending_peers, start_time, ...} to decide if we need to send_check_trim_status again.